qa/L0_memory_growth/test.sh

```diff
@@ -70,6 +70,16 @@ INSTANCE_CNT=2
 CONCURRENCY=32
 CLIENT_BS=8
 
+# Set the number of repetitions in nightly and weekly tests
+# Set the email subject for nightly and weekly tests
+if [ "$TRITON_WEEKLY" == 1 ]; then
+    REPETITION=240
+    EMAIL_SUBJECT=1
+else
+    REPETITION=3
+    EMAIL_SUBJECT=0
```

Comment:
Maybe changing the script to take a string as suffix to the subject and then setting `EMAIL_SUBJECT` to "daily" / "weekly" here, or changing the variable name to be "USE_WEEKLY_EMAIL_SUBJECT" so it matches the boolean usage.

Reply:
Will modify.

Reply:
Scripts are updated.